projects/1.20/assets/sons-of-sins/sons_of_sins/lang/zh_cn.json

```
@@ -0,0 +1,210 @@
+{
+  "advancements.a_heal_stick.descr": "使用装满骨灰的以太剑",
+  "advancements.a_heal_stick.title": "治愈法杖",
+  "advancements.ash_diamond.descr": "在血液中投掷16个灰烬",
+  "advancements.ash_diamond.title": "骨灰钻石",
+  "advancements.freddys_favorite_alloy.descr": "做出灵魂钢",
+  "advancements.freddys_favorite_alloy.title": "Freddy's 最爱的合金",
+  "advancements.ghost_like_paxel.descr": "使用装满骨灰的塑形器",
+  "advancements.ghost_like_paxel.title": "鬼一样的Paxel?",
```

```suggestion
  "advancements.ghost_like_paxel.title": "鬼一样的Paxel？",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。